src/bindgen/ir/documentation.rs

```diff
@@ -76,7 +76,12 @@ impl Source for Documentation {
             _ => (),
         }
 
-        for line in &self.doc_comment {
+        let end = match config.documentation_length {
```

Comment:
Can we handle it also in the cython case just for completeness? Should be trivial moving this a few lines above but if you don't have the time I'm happy to do it.